backend/src/main/scala/com/softwaremill/bootzooka/util/SecureRandomId.scala

```diff
@@ -0,0 +1,48 @@
+package com.softwaremill.bootzooka.util
+
+import cats.effect.Sync
+import java.security.SecureRandom
+import org.apache.commons.codec.binary.Hex
+
+/** copy-pasted from TSec https://github.com/jmcardon/tsec */
```

Comment:
Not sure, if it :
* should stay as it is 
or 
* be cut in "one class for file" manner and placed in separate package name like `securerandom` or similar.